src/App.vue

```diff
@@ -240,6 +240,8 @@ export default {
 	},
 
 	beforeMount() {
+		this.migrateDirectLocalStorageToNextcloudBrowserStorage()
+
```

Comment:
`migrateDirectLocalStorageToNextcloudBrowserStorage` is not a vue component method. It doesn't use any component's properties, it is not used as a component's property. It should be a function outside of the component.

A small drawback of using it as a component method is that the very first time it is called is only after the root application component is already initialized and created. So we may lose the value if it is got from the storage before (vuex store init, global application init, most of other imported ES modules).

Suggestion: https://github.com/nextcloud/spreed/blob/master/src/init.js

Reply:
Like this?